docs/changelog.rst

```diff
@@ -1,6 +1,26 @@
 Changelog
 =========
 
+0.3.0 (Unreleased)
+------------------
+
+Changed
+~~~~~~~
+
+- Refactor to build upon `ipython-sql <https://pypi.org/project/ipython-sql/>`__.
+  Removes several functions that can be replaced with direct usage of ipython-sql magics in the notebook,
+  and replace all remaining sql calls with calls to ipython-sql.
```

Comment:
Put paragraph text on a single line. It's cumbersome to edit a paragraph that's split on multiple lines, and text editors can do word wrapping.